mod/metric_answer_lookup/lib/answer/answer_class_methods.rb

```diff
@@ -123,5 +123,14 @@ def latest_year metric_id, company_id
     def answered? metric_id, company_id
       where(metric_id: metric_id, company_id: company_id).exist?
     end
+
+    def unknown? val
+      val.to_s.casecmp("unknown").zero?
+    end
+
+    def find_by_answer_id answer_id
+      return unless answer_id
+      super
+    end
```

Comment:
without this, new cards were looking up answers that had nil answer ids (and finding them!)